src/TSTransformer/util/jsx/getJsxIndex.ts

```diff
@@ -8,12 +8,19 @@ function getJsxIndex(factory: string) {
 	return propertyAccessExpressionChain(luau.id(namespace), path);
 }
 
-export function getCreateFragmentIndex(state: TransformState) {
-	assert(state.compilerOptions.jsxFragmentFactory);
+export function getJsxCreateFragmentIndex(state: TransformState) {
+	if (!state.compilerOptions.jsxFragmentFactory) {
```

Comment:
This should also validate for being a valid luau identifier - can currently cause a compiler crash by setting it to `$React`

Reply:
I believe this is side stepped in the latest changes by using transformEntityName